packages/deploy/hardhat.config.ts

```diff
@@ -300,58 +300,106 @@ const namedAccounts = {
   },
 };
 
+export enum DEPLOY_NETWORKS {
+  HARDHAT = 'hardhat',
+  ETH_SEPOLIA = 'sepolia',
+  ETH_GOERLI = 'goerli',
+  EHT_MAINNET = 'mainnet',
+  BSC_TESTNET = 'bscTestnet',
+  BSC_MAINNET = 'bscMainnet',
+  AMOY = 'amoy',
+  POLYGON = 'polygon',
+  MUMBAI = 'mumbai',
+  BASE_SEPOLIA = 'baseSepolia',
+  BASE_MAINNET = 'base',
+}
+
 /**
  * TAGS:
- *  - mainnet -> production networks (you must pay for gas!!!)
- *  - L1      -> Layer 1 networks
- *  - L2      -> Layer 2 networks
+ *  - mainnet -> production DEPLOY_NETWORKS (you must pay for gas!!!)
+ *  - L1, L1-prod, L1-test -> Layer 1 DEPLOY_NETWORKS
+ *  - L2, L2-prod, L2-test -> Layer 2 DEPLOY_NETWORKS
  */
+
 const networks = {
-  hardhat: {
-    tags: ['L1', 'L2'],
+  [DEPLOY_NETWORKS.HARDHAT]: {
+    tags: ['L1', 'L1-prod', 'L1-test', 'L2', 'L2-prod', 'L2-test'],
```

Comment:
We should define for each network which tag(s) apply, since it isn't obvious in the enum list (and probably also not from our current usage). 
We would also need to review all existing scripts in `packages/deploy` to assess which new tags (test / prod) apply.